app/controllers/api/databases.php

```diff
@@ -101,13 +101,13 @@ function createAttribute(string $databaseId, string $collectionId, Document $att
     $default = $attribute->getAttribute('default');
     $options = $attribute->getAttribute('options', []);
 
-    $db = Authorization::skip(fn () => $dbForProject->getDocument('databases', $databaseId));
+    $database = $dbForProject->getDocument('databases', $databaseId);
```

Comment:
Why do we remove the skip here?

Reply:
This method can only be called with an API key, which already disables auth